ast/ddl.go

```diff
@@ -3426,3 +3449,68 @@ func (n *FlashBackTableStmt) Accept(v Visitor) (Node, bool) {
 	}
 	return v.Leave(n)
 }
+
+type PlacementActionType int
+
+const (
+	PlacementAdd PlacementActionType = iota + 1
+)
+
+type PlacementRole int
+
+const (
+	PlacementRoleFollower PlacementRole = iota + 1
+	PlacementRoleLeader
+	PlacementRoleLearner
+	PlacementRoleVoter
+)
+
+type PlacementSpec struct {
+	node
+
+	Tp     PlacementActionType
+	Labels string
+	Role   PlacementRole
+	Count  uint64
+}
+
+func (n *PlacementSpec) Restore(ctx *format.RestoreCtx) error {
+	switch n.Tp {
+	case PlacementAdd:
+		ctx.WriteKeyWord("ADD PLACEMENT ")
+	default:
+		return errors.Errorf("invalid PlacementActionType: %d", n.Tp)
+	}
+
+	ctx.WriteKeyWord("LABEL")
+	ctx.WritePlainf("=\"%s\" ", n.Labels)
```

Comment:
```suggestion
	ctx.WritePlain("=")
	ctx.WriteString(n.Labels)
```

you don't want to restore to an invalid string if the label contains a `'` or `"`.

Reply:
Thanks.